app/build.gradle

```diff
@@ -146,10 +146,12 @@ dependencies {
     implementation project(':ui:base-app')
     implementation project(':ui:shortcuts')
     implementation project(':ui:tract-renderer')
+    implementation project(':ui:tutorial-renderer')
```

Comment:
`ui:tutorial`, `renderer` is specific to tool renderers

Reply:
@frett I am still working on this PR and I am going to clean up the Gradle files on my final pass.

Reply:
although since it sounds like there are 2 different sets of tutorial screens, this could be fine for the module name